samples/nextjs/scripts/disconnected-mode-proxy.ts

```diff
@@ -9,20 +9,22 @@
 
 /* eslint-disable no-console */
 
-const fs = require('fs');
-const path = require('path');
-const { createDefaultDisconnectedServer } = require('@sitecore-jss/sitecore-jss-dev-tools');
-const config = require('../package.json').config;
+import fs from 'fs';
+import path from 'path';
+import { createDefaultDisconnectedServer } from '@sitecore-jss/sitecore-jss-dev-tools';
+import { config } from '../package.json';
 
 const touchToReloadFilePath = 'src/temp/config.js';
 
 const serverOptions = {
   appRoot: path.join(__dirname, '..'),
   appName: config.appName,
+  // Prevent require of ./sitecore/definitions/config.js, becuase ts-node is running
+  requireArg: '',
   watchPaths: ['./data'],
   language: config.language,
```

Comment:
You can't simply remove `language` from `package.json` because sitecore-jss-cli look at this value.
https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-cli/src/scripts/manifest.ts#L106-L115